python/ray/_private/runtime_env/path_utils.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+from typing import List, Optional
+from ray._private.runtime_env import env_utils
+
+
+INTERNAL_PIP_FILENAME = "ray_runtime_env_internal_pip_requirements.txt"
+MAX_INTERNAL_PIP_FILENAME_TRIES = 100
+
+
+class PathHelper:
+    @staticmethod
+    def get_virtualenv_path(target_dir: str) -> str:
+        return os.path.join(target_dir, "virtualenv")
+
+    @classmethod
+    def get_virtualenv_python(cls, target_dir: str) -> str:
+        virtualenv_path = cls.get_virtualenv_path(target_dir)
+        if env_utils._WIN32:
+            return os.path.join(virtualenv_path, "Scripts", "python.exe")
+        else:
+            return os.path.join(virtualenv_path, "bin", "python")
+
+    @classmethod
+    def get_virtualenv_activate_command(cls, target_dir: str) -> List[str]:
+        virtualenv_path = cls.get_virtualenv_path(target_dir)
+        if env_utils._WIN32:
+            cmd = [os.path.join(virtualenv_path, "Scripts", "activate.bat")]
+
+        else:
+            cmd = ["source", os.path.join(virtualenv_path, "bin/activate")]
+        return cmd + ["1>&2", "&&"]
```

Comment:
Feel these should also belong to `env_utils.py` since they are all about virtualenv?

Reply:
Sure, updated.